docs/maintainer/understanding_conda_forge/feedstocks.md

```diff
@@ -0,0 +1,184 @@
+# The feedstock
+
+A feedstock is the home of one recipe; it may produce more than one conda artifact in the case of a multi-output recipe.
+The feedstock is the stage for most events about a recipe.
```

Comment:
I wonder if the use of the word "stage" here, while correct, may confuse some reads. After all, we have a repo named staged-recipes.

BTW, do you believe that for linearity's sake we should start with staged-recipe? I don't have a strong opinion b/c that argument is weak if you are a new contributor to an existing recipe. They the first entry is a feedstock.

Reply:
The suggested reading order is the one in sidebar, so yes, staged-recipes goes first:

<img width="364" alt="image" src="https://github.com/user-attachments/assets/15b7337b-5c22-4b23-9423-4fce391da437">